lib/classes/Database.js

```diff
@@ -1,10 +1,10 @@
-const {Sequelize} = require("sequelize");
+const { Sequelize } = require("sequelize");
 const Guild = require("../database/models/Guild.js");
 const GuildChannel = require("../database/models/GuildChannel.js");
 const GuildMember = require("../database/models/GuildMember.js");
 const User = require("../database/models/User.js");
-const fs = require("fs").promises;
-const {log} = require("../classes/Utils.js");
+const { log } = require("../classes/Utils.js");
```

Comment:
I feel like our IDEs using different configuration for spaces between object members and braces. Can you add this setting to editorconfig for JetBrain's IDE?

`ij_javascript_spaces_within_object_literal_braces = true`

This will force my IDE to set this spaces.

Reply:
Why not add this setting to an editor-specific file in .idea folder?

Reply:
Commiting IDEA configuration not that good idea. This directory have a lot of garbage and need some time to select correct files without any system-specific configuration values. So while I'm not made this, we can use this IDE-specific option in editorconfig. You can also write TODO comment for this.

Reply:
Will do it later.